drivers/health_flexible_service/app/models/health_flexible_service/member_expiration.rb

```diff
@@ -0,0 +1,39 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HealthFlexibleService
+  class MemberExpiration
+    include ArelHelper
+
+    def self.viewable_by(user)
+      GrdaWarehouse::WarehouseReports::ReportDefinition.where(url: url).
+        viewable_by(user).exists?
+    end
+
+    def self.url
+      'health_flexible_service/warehouse_reports/member_expiration'
+    end
+
+    def title
+      'VPR Member Expiration'
+    end
+
```

Comment:
Do you use any of this, or is it just left over from a report with PDF download?